packages/react-inbox/src/components/Messages2.0/Header.tsx

```diff
@@ -35,9 +35,9 @@ const Container = styled.div<{ view?: string }>(({ theme }) => {
       fontSize: 16,
       fontWeight: 700,
       height: "42px",
-      color: "rgb(36, 50, 75)",
-      backgroundColor: "#F2F6F9",
-      borderBottom: "1px solid rgb(222, 232, 240)",
+      backgroundColor: "var(--ci-background)",
```

Comment:
minor thing, this should probably be `background` instead of backgroundColor in the case that someone passes in shorthand css for `background`